application/views/user_views.py

```diff
@@ -0,0 +1,124 @@
+from werkzeug.security import check_password_hash
+from flask_jwt_extended import jwt_required, get_raw_jwt, create_access_token, get_jwt_identity
+from werkzeug.security import generate_password_hash
+from flask_restplus import Resource, Namespace, fields
+from flask import request, jsonify
+from datetime import datetime
+import re
+
+from . import blacklist
+
+from application.models.user_model import User
+from application import db
+
+
+
+try:
+    # Python 3
+    from urllib.parse import urlparse, parse_qs
+except ImportError:
+    # Python 2
+    from urlparse import urlparse, parse_qs
+
+api = Namespace('Users', Description='User operations')
+
+def validate_email(email):
+    match = re.match(r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+\.[a-zA-Z0-9.]*\.*[com|org|edu]{3}$)",email)
+    if match is not None:
+        return True
+    return False 
+
+class UserSignUp(Resource):
+    """ Enable user to sign up """
+    def post(self):
+        
+        userData = request.get_json()
```

Comment:
This variable is not named in a pythonic way.

Reply:
What happens if a key is missing in this userData dictionary? Please handle these edge cases.